### ISSUE TITLE
Stuck forever in update

### ISSUE THREAD

**Issue by XaverStiensmeier** [BiBiServ/bibigrid#484]
Currently, when locations have a really weak connection, it can happen that BiBiGrid seems to be stuck when in reality the initial update of the server just takes forever. BiBiGrid displays no warning and Ansible will be stuck at the update task.

**Comment by XaverStiensmeier** [comment]
This might be "fixed" now by explicitly waiting on that lock in another task. However, this still needs to be tested.

**Comment by XaverStiensmeier** [comment]
The real reason probably has been the faster cloud-init service that hindered the execution until the wait for service routine in ansible. Solved by #500 in the master and #501 in the dev branch. Also, explicitly waiting (see above) has been kept as an additional security mechanism.
